src/gui/EditWidgetIcons.h

```diff
@@ -74,6 +74,9 @@ private Q_SLOTS:
     Database* m_database;
     Uuid m_currentUuid;
     QString m_url;
+    QUrl redirectUrl;
```

Comment:
These new member variables do not conform to the coding requirements. Member variables should be preceded with m_ to differentiate from local/stack variables.